controllers/user.go

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import (
+	"fmt"
+	forms "go-crawler-challenge/forms/user"
+	"net/http"
+
+	log "github.com/beego/beego/v2/core/logs"
+	beego "github.com/beego/beego/v2/server/web"
+)
+
+//  UserController operations for User
+type UserController struct {
+	beego.Controller
+}
+
+// URLMapping ...
+func (c *UserController) URLMapping() {
+	c.Mapping("New", c.New)
+	c.Mapping("Create", c.Create)
+}
+
+// New handles a form for creating a new user
+// @Title New
+// @Description show a new user form
+// @Success 200
+// @router / [get]
+func (c *UserController) New() {
+	c.Layout = "layouts/application.html"
+	c.TplName = "user/new.html"
+}
+
+// Create handles validation and adding a new unique user
+// @Title Create
+// @Description create a new unique user
+// @Success 302 redirect to the signup page
+// @Failure 302 edirect to the signup page and print some error messages
+// @router / [post]
+func (c *UserController) Create() {
+	form := forms.RegistrationForm{}
+
+	err := c.ParseForm(&form)
+	if err != nil {
+		log.Info(fmt.Sprintf("%v", err.Error()))
+	}
+
+	_, errors := form.Create()
```

Comment:
Could we use the name variable name `err` here instead of `errors` for consistency?

Reply:
@junan This `errors` is the set of error returning from here https://github.com/Lahphim/go-crawler-challenge/pull/36/files/9cb1f3eb789c7027839cb37c7374d7c183f58b3c#diff-725bfeda29aae6641375cec7b986115cb233155976a3bfe6f294a9253305f4c6R35 🤔